content/hi/docs/reference/glossary/application-architect.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: एप्लीकेशन आर्किटेक्ट (Application Architect)
```

Comment:
It would be beneficial if changes to existing glossary files could be addressed separately from this pull request. This pull request for localizing the glossary into Hindi is already quite large, with over 100+ files to review. If possible, breaking it down into smaller pull requests would make it easier and more likely to be reviewed promptly.

Reply:
Should disperse them according to their tags?

Reply:
![Screenshot from 2024-02-18 18-42-15](https://github.com/kubernetes/website/assets/96516301/bfb7d987-9d92-42ed-b05d-0d997d8cda98)

Reply:
As, there are many glossary terms that are mentioned in Hindi, would it be better to keep them in Hindi or English?

Reply:
![Screenshot from 2024-02-18 18-47-56](https://github.com/kubernetes/website/assets/96516301/58aa7203-6f64-4dee-a9a0-31c96064a7cf)